tests/ha_cases/run.sh

```diff
@@ -175,14 +184,36 @@ function test_kill_worker() {
     run_dm_worker $WORK_DIR/worker4 $WORKER4_PORT $cur/conf/dm-worker4.toml
     check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER4_PORT
 
-    echo "kill dm-worker3"
+    echo "restart dm-worker3"
     ps aux | grep dm-worker3 |awk '{print $2}'|xargs kill || true
     check_port_offline $WORKER3_PORT 20
     rm -rf $WORK_DIR/worker3/relay_log
 
     echo "wait and check task running"
     check_http_alive 127.0.0.1:$MASTER_PORT/apis/${API_VERSION}/status/test '"name":"test","stage":"Running"' 10
 
+    run_dm_worker $WORK_DIR/worker3 $WORKER3_PORT $cur/conf/dm-worker3.toml
+    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER3_PORT
+
+    echo "isolate dm-worker4"
```

Comment:
should we need to update the function name from `test_kill_worker` to another one with `isolate`?